ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_approved-decision.js

```diff
@@ -0,0 +1,175 @@
+/// <reference types ='Cypress'/>
+
+// uri to be updated once academisation API is integrated
+let url = Cypress.env('url') + '/task-list/2050?rd=true'
+
+describe('103195 Approved record decision', ()=> {
+    
+    beforeEach(() => {
+        cy.visit(url)
+        cy.clearCookies()
+
+    })
+    // Approve Happy Path journey
+    // *** A new project is need to run tests - skipping for now
+    it.skip('TC01: J1 - Navigate to project to record a decision', () => {
+        cy.get('[id="record-decision-link"]').click()
```

Comment:
Small thing: To be honest it doesn't really make much difference but for id's you can use the selector `#record-decision-link`

Reply:
yes, I could but it's not best practice to call it via '#' https://docs.cypress.io/guides/references/best-practices

Reply:
It looks like this article would rather we added data attributes to elements rather than using id's which could change for style reasons. I'll remember that for next time.